src/rfquack_radio.h

```diff
@@ -300,33 +293,36 @@ void rfquack_rx_loop() {
     return;
 
   rfquack_Packet pkt;
-  uint8_t len = sizeof(pkt.data.bytes);
+  uint8_t len = rfquack_rf->getPacketLength();
```

Comment:
Just make sure that this is cached somewhere, so that we don't generate too much SPI traffic.